packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,47 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) file system to a local directory. 
+
+The following sample enables the lambda function to mount the filesystem from Amazon EFS via the `accessPoint` to 
+the local `/mnt/msg` directory.
+
+```ts
+const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
+  fileSystem,
+  createAcl: {
+    ownerGid: '1000',
+    ownerUid: '1000',
+    permissions: '755',
+  },
+  path: '/lambda',
+  posixUser: {
+    uid: '1000',
+    gid: '1000',
+  },
```

Comment:
Is this the minimal usage? I would expect most of these to have sensible defaults and then this example can and should be much more concise. This is not about access points, it's about lambda.

Reply:
Agree. But unfortunately I didn't find any document for the recommended permission settings in this case. However, according the our announcement blog post - [New – A Shared File System for Your Lambda Functions](https://aws.amazon.com/tw/blogs/aws/new-a-shared-file-system-for-your-lambda-functions/), we use `1001` for both owner/posix uid/gid and `750` as the permission. How about use this settings as the defaults ?

Reply:
Hi @eladb 

From my current proposed design, `lambda.FileSystem.fromEfsAccessPoint()` requires `accessPoint: efs.IAccessPoint` as the only property.

We can update the sample in the README to create the accessPoint like this

```
const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
  fileSystem,
  path: '/lambda',
});
```
And create the function like this

```ts
const fn = new lambda.Function(stack, 'MyLambda', {
  code,
  handler,
  runtime,
  vpc,
  filesystems: {
    filesystem: lambda.FileSystem.fromEfsAccessPoint(accessPoint),
    localMountPath: '/mnt/msg',
  },
});
```
This might look more concise. What do you think?

Reply:
@nija-at what do you think? Should we reach out to lambda/EFS to consult about defaults?

Reply:
@eladb - I believe we landed at some decent defaults here - https://github.com/aws/aws-cdk/pull/8631. @pahud did some deep dive into what EFS defaults to.

`path` is optional and defaults to `/`. 
`createAcl` is optional and is only needed if the location defined by `path` needs to be created as part of mounting the access point.
`posixUser` is optional and is only needed when [user enforcement](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html#enforce-identity-access-points) should be enabled.

Reply:
So lets use those defaults.

Reply:
> Hi @eladb
> 
> From my current proposed design, `lambda.FileSystem.fromEfsAccessPoint()` requires `accessPoint: efs.IAccessPoint` as the only property.
> 
> We can update the sample in the README to create the accessPoint like this
> 
> ```
> const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
>   fileSystem,
> });
> ```
> 
> And create the function like this
> 
> ```ts
> const fn = new lambda.Function(stack, 'MyLambda', {
>   code,
>   handler,
>   runtime,
>   vpc,
>   filesystems: {
>     filesystem: lambda.FileSystem.fromEfsAccessPoint(accessPoint, '/mnt/msg'),
>   },
> });
> ```
> 
> This might look more concise. What do you think?

Yes this is what I'd like to see